### DISCUSSION TITLE
WCAG Compliance

### DISCUSSION THREAD

**Discussion by dahegit** [ant-design-blazor/ant-design-blazor#1954] [General]
Again, Ant-design-blazor is awesome! Thank you!

I there a roadmap for implementing WCAG Compliance?
Some of the issues I have encountered is

- Dublicated elemen id:s
- Missing title / name attribut on buttons and links
- Not been able to use the tab key to select and element and then Enter to click it

**Comment by ElderJames** [comment]
Thanks for contacting us @dahegit .

We would very much like to see improvements in i11y as well. But because we have so few contributors at the moment, we are not yet able to take this aspect into account. If you are interested, please feel free to submit PRs, we would be very grateful!
